pkg/autoscaler/scaling/autoscaler_test.go

```diff
@@ -469,10 +468,13 @@ func TestAutoscalerRateLimitScaleDown(t *testing.T) {
 	expectScale(t, a, time.Now(), ScaleResult{1, expectedEBC(10, 61, 1, 10), na, true})
 }
 
-func eraseEndpoints() {
-	ep, _ := fake.KubeClient.CoreV1().Endpoints(fake.TestNamespace).Get(fake.TestService, metav1.GetOptions{})
-	fake.KubeClient.CoreV1().Endpoints(fake.TestNamespace).Delete(fake.TestService, nil)
-	fake.KubeInformer.Core().V1().Endpoints().Informer().GetIndexer().Delete(ep)
+func TestCantCountPods(t *testing.T) {
```

Comment:
This seems duplicated with the function below.

Reply:
some weird merge